src/reporter/index.js

```diff
@@ -188,7 +190,7 @@ export default class Reporter {
 
             if (!reportItem.pendingStarts) {
                 if (this.plugin.reportTestStart) {
-                    const testStartInfo = { testRunIds: reportItem.testRunIds };
+                    const testStartInfo = { testRunIds: reportItem.testRunIds, testId: reportItem.test.id };
```

Comment:
We need access to `browserInfo` in case if no actions were executed before an error (currently we get it in the `reportTestActionDone` method). I suggest matching 'testRunId' with `browserInfo` here.